examples/language_model/bloom/run_eval.py

```diff
@@ -0,0 +1,323 @@
+# Copyright (c) 2023 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import annotations
+
+import json
+import math
+import re
+import time
+from pprint import pprint as print
+
+import numpy as np
+import paddle
+from args import get_parser
+from configuration import BloomConfig
+from modeling import BloomForPretraining
+from paddle.io import DataLoader
+from transformers import AutoTokenizer
```

Comment:
from paddlenlp.transformers import AutoTokenizer

Reply:
it will be updated tomorrow.